### PR TITLE
Fix various formatting issues in `osu! Philippines: CONQuest Festival 2023` news post and related wiki article

### PR DESCRIPTION
**Pull request by Walavouchey**

### CODE REVIEW

File: news/2023/2023-05-29-osu-philippines-conquest-festival-2023.md [hunk 1]
```diff
@@ -4,26 +4,26 @@ title: "osu! Philippines: CONQuest Festival 2023"
...
-![Banner](/wiki/shared/news/2023-05-29-osu-philippines-conquest-festival-2023/banner.png)
+![](/wiki/Tournaments/oPQS/2/img/banner.png)
 
-The osu! booth will have games, goodies, and more as the Filipino community; players and non-players alike, get to know and enjoy osu! more!
+The osu! booth will have games, goodies, and more as the Filipino community, players and non-players alike, get to know and enjoy osu! more!
 
 ## Tournament
 
-The main event of the space is the **osu! Philippines Quickfire Showdown 2023**. A lineup of confirmed players will go through a LAN tournament featuring a *Super Smash Bros.*-styled Brawl format. The tournament will take place on Day 1 and Day 2, with prizes in store!
+The main event of the space is the **osu! Philippines Quickfire Showdown 2023**. A lineup of confirmed players will go through a LAN tournament featuring a *Super Smash Bros.*–tyled brawl format. The tournament will take place on day 1 and day 2, with prizes in store!
```
Comment (pishifat) [inline news/2023/2023-05-29-osu-philippines-conquest-festival-2023.md:15]: missing s in style

File: news/2023/2023-05-29-osu-philippines-conquest-festival-2023.md [hunk 2]
```diff
@@ -4,26 +4,26 @@ title: "osu! Philippines: CONQuest Festival 2023"
...
 
 For more information about the tournament, refer to [this wiki article](/wiki/Tournaments/oPQS/2).
 
-## Booth Activties
+## Booth activties
 
-The tournament isn't the only thing that's in store for the attendees! If the burning flames of the competition is not their thing, other activities can be done: for community members that want to meet up with their friends or for those meeting for the first time, this is their chance to create new memories and friends at the booth's Player Hub! Newcomers and those already familiar with the game will be able to also play osu!(lazer) at the booth as well.
+The tournament isn't the only thing that's in store for the attendees! If the burning flames of the competition is not to their thing, there are other activities as well: for community members that want to meet up with their friends or for those meeting for the first time, this is their chance to create new memories and friends at the booth's Player Hub! Newcomers and those already familiar with the game will also be able to play osu!(lazer) at the booth as well.
```
Comment (pishifat) [inline news/2023/2023-05-29-osu-philippines-conquest-festival-2023.md:21]: > If the burning flames of the competition is not to their thing

original sounds more natural. "not your/their thing" is a pretty common casual phrase

File: wiki/Tournaments/oPQS/2/en.md [hunk 3]
```diff
@@ -176,14 +176,15 @@ The osu! Philippines Quickfire Showdown 2023 was ran by various community member
 - Double Time
   1. [Amatsuki - DORAEMON [Insane]](https://osu.ppy.sh/beatmapsets/853519#osu/1783814)
   2. [Sound Souler - Eternalism 2 [Gibune's Insane]](https://osu.ppy.sh/beatmapsets/1299190#osu/2694599)
+  2. [Sound Souler - Eternalism 2 [Gibune's Insane]](https://osu.ppy.sh/beatmapsets/1299190#osu/2694599)
   3. [mafumafu - Kakushigoto [NiNo's Light Insane.]](https://osu.ppy.sh/beatmapsets/770023#osu/1618984)
   4. [James Landino - Reaction feat. Slyleaf [Milan-'s Insane]](https://osu.ppy.sh/beatmapsets/972816#osu/2036668)
 - Tiebeaker
-  1. [Amatsuki - Chiisana Koi no Uta [Together]](https://osu.ppy.sh/beatmapsets/534788#osu/1169701)
+  1. *[Amatsuki - Chiisana Koi no Uta [Together]](https://osu.ppy.sh/beatmapsets/534788#osu/1169701)*
```
Comment (pishifat) [inline wiki/Tournaments/oPQS/2/en.md:182]: is this supposed to be bold like the other tiebreakers?

**Review by pishifat — CHANGES_REQUESTED** [review]

File: news/2023/2023-05-29-osu-philippines-conquest-festival-2023.md (hunk 2 above)
Comment (Walavouchey) [reply news/2023/2023-05-29-osu-philippines-conquest-festival-2023.md:21]: oh i somehow misread "thing" as "liking". you're entirely correct

```suggestion
The tournament isn't the only thing that's in store for the attendees! If the burning flames of the competition is not their thing, there are other activities as well: for community members that want to meet up with their friends or for those meeting for the first time, this is their chance to create new memories and friends at the booth's Player Hub! Newcomers and those already familiar with the game will also be able to play osu!(lazer) at the booth as well.
```

File: news/2023/2023-05-29-osu-philippines-conquest-festival-2023.md (hunk 1 above)
Comment (Walavouchey) [reply news/2023/2023-05-29-osu-philippines-conquest-festival-2023.md:15]: rushed bc i wanna slep

```suggestion
The main event of the space is the **osu! Philippines Quickfire Showdown 2023**. A lineup of confirmed players will go through a LAN tournament featuring a *Super Smash Bros.*–styled brawl format. The tournament will take place on day 1 and day 2, with prizes in store!
```

File: wiki/Tournaments/oPQS/2/en.md (hunk 3 above)
Comment (Walavouchey) [reply wiki/Tournaments/oPQS/2/en.md:182]:
```suggestion
  1. **[Amatsuki - Chiisana Koi no Uta [Together]](https://osu.ppy.sh/beatmapsets/534788#osu/1169701)**
```

File: news/2023/2023-05-29-osu-philippines-conquest-festival-2023.md [hunk 4]
```diff
@@ -4,26 +4,26 @@ title: "osu! Philippines: CONQuest Festival 2023"
...
 
 ## Tournament
 
-The main event of the space is the **osu! Philippines Quickfire Showdown 2023**. A lineup of confirmed players will go through a LAN tournament featuring a *Super Smash Bros.*-styled Brawl format. The tournament will take place on Day 1 and Day 2, with prizes in store!
+The main event of the space is the **osu! Philippines Quickfire Showdown 2023**. A lineup of confirmed players will go through a LAN tournament featuring a *Super Smash Bros.*–styled brawl format. The tournament will take place on day 1 and day 2, with prizes in store!
 
 For more information about the tournament, refer to [this wiki article](/wiki/Tournaments/oPQS/2).
 
-## Booth Activties
+## Booth activties
```
Comment (pishifat) [inline news/2023/2023-05-29-osu-philippines-conquest-festival-2023.md:19]:
```suggestion
## Booth activities
```

**Review by pishifat — APPROVED** [review]

**Pull request merged**